pkg/sdn/plugin/node.go

```diff
@@ -121,6 +162,36 @@ func (node *OsdnNode) Start() error {
 	return nil
 }
 
+// FIXME: this should eventually go into kubelet via a CNI UPDATE/CHANGE action
+// See https://github.com/containernetworking/cni/issues/89
+func (node *OsdnNode) UpdatePod(namespace string, name string, id kubeletTypes.ContainerID) error {
+	const pluginPath = "/opt/cni/bin/openshift-sdn"
```

Comment:
This doesn't seem like it should be a constant, especially not declared here.  Declare it on the node and initialize it or pass it in.